install/resources/settings.xml

```diff
@@ -876,6 +876,30 @@ min=0]]></optionscode>
 			<settingvalue><![CDATA[0]]></settingvalue>
 			<isdefault>1</isdefault>
 		</setting>
+		<setting name="threadviews_countspiders">
+			<title>Increment Thread View Count on Spider Visits?</title>
+			<description><![CDATA[Here you can include views by spiders in the thread view counter.]]></description>
+			<disporder>15</disporder>
+			<optionscode><![CDATA[yesno]]></optionscode>
+			<settingvalue><![CDATA[0]]></settingvalue>
+			<isdefault>0</isdefault>
```

Comment:
I believe `isdefault` should be `1` for all core settings

Reply:
I will fix in 9 hours. I was confused with this, not properly documented.

Thx

Reply:
Changed.